lib/edition_dependencies_populator.rb

```diff
@@ -5,8 +5,8 @@ def initialize(edition)
   end
 
   def populate!
-    @edition.depended_upon_contacts = Govspeak::ContactsExtractor.new(@edition.body).contacts
-    @edition.depended_upon_editions = Govspeak::DependableEditionsExtractor.new(@edition.body).editions
+    @edition.depended_upon_contacts = Govspeak::ContactsExtractor.new(@edition.body).contacts.uniq
```

Comment:
@heathd: i thought about doing that too. however, `ContactsExtractor` in its earlier form in the `GovspeakHelper` had a test for ensuring that duplicates are not uniq'd, which i ported that as-is. in the context of govspeak, i think preserving duplicates makes sense, because it might be valuable to know the frequency of occurrence of a particular embedded element. in the context of creating dependencies, it doesn't make sense to have duplicates, so we uniq the result.

i'm not too sure about either approaches, so if you still feel it'll be good to push the responsibility of returning a uniq array to the extractors, i'll change that. let me know.